src/main/kotlin/org/jetbrains/webstorm/ide/colors/WebAssemblyColorSettingPage.kt

```diff
@@ -0,0 +1,37 @@
+package org.jetbrains.webstorm.ide.colors
+
+import com.intellij.openapi.editor.colors.TextAttributesKey
+import com.intellij.openapi.fileTypes.SyntaxHighlighter
+import com.intellij.openapi.options.colors.AttributesDescriptor
+import com.intellij.openapi.options.colors.ColorDescriptor
+import com.intellij.openapi.options.colors.ColorSettingsPage
+import org.jetbrains.webstorm.ide.WebAssemblySyntaxHighlighter
+import org.jetbrains.webstorm.ide.icons.WebAssemblyIcons
+import javax.swing.Icon
+
+class WebAssemblyColorSettingPage : ColorSettingsPage {
```

Comment:
do we really need it at this moment?

Reply:
Should I remove it? We don't need it now but It was kinda free.

Reply:
if it works — ok, but we definitely need to fix the code sample (getDemoText) below

Reply:
Yes, I'll fix this moment